components/config-mgmt-service/integration_test/node_export_test.go

```diff
@@ -25,6 +25,81 @@ import (
 	"time"
 )
 
+func TestNodeExportMissingTags(t *testing.T) {
```

Comment:
nitpick: "Node export missing tags" describes the bug you're fixing, but not the behavior you're testing. Something like "Node export includes chef tags" describes the desired behavior.